docs/intro/upgrading-to-ionic-6.md

```diff
@@ -0,0 +1,443 @@
+# Upgrading to Ionic 6
+
+Follow this guide to upgrade your Ionic 5 apps to Ionic 6.
+
+## Getting Started
+
+### Angular
+
+1. Ionic 6 supports Angular 12+. Update to the latest version of Angular by following the [Angular Update Guide](https://update.angular.io/).
+2. Update to the latest version of Ionic 6:
+
+```shell
+npm install @ionic/angular@6
+```
+
+If you are using Ionic Angular Server, be sure to update that as well:
+
+```shell
+npm install @ionic/angular@6 @ionic/angular-server@6
+```
+
+3. Remove any usage of `Config.set()`. Instead, set your config in `IonicModule.forRoot()`. See the [Angular Config Documentation](../angular/config) for more examples.
+4. Remove any usage of the `setupConfig` function previously exported from `@ionic/angular`. Set your config in `IonicModule.forRoot()` instead.
+
+### React
+
+1. Ionic 6 supports React 17+. Update to the latest version of React:
+
+```shell
+npm install react@latest react-dom@latest
+```
+
+2. Update to the latest version of Ionic 6:
+
+```shell
+npm install @ionic/react@6 @ionic/react-router@6
+```
+
+3. Update the `test` field in the `scripts` object of your `package.json` to include `transformIgnorePatterns`:
+
+```json
+"scripts": {
+  "test": "react-scripts test --transformIgnorePatterns 'node_modules/(?!(@ionic/react|@ionic/react-router|@ionic/core|@stencil/core|ionicons)/)'",
+  ...
+}
+```
```

Comment:
Could we use a different syntax highlighting format to avoid the weird red highlight on `...`? (Also applies in a few other spots)

Reply:
Is the red highlight showing up on the docs preview or just in this PR preview?

Reply:
Ahh I see, I was looking at the MD render in Github. All good on the actual site 👍